include/zephyr/net/wifi_mgmt.h

```diff
@@ -752,6 +752,10 @@ struct wifi_twt_params {
 			 * prepare the data before TWT SP starts.
 			 */
 			uint32_t twt_wake_ahead_duration;
+			/** TWT info enabled or disable */
```

Comment:
can you please elaborate? there is a separate `TWT Information frame` action frame.

Reply:
Yes, it is.